src/main/java/com/julienviet/pgclient/impl/PostgresSQLConnection.java

```diff
@@ -30,81 +38,108 @@ public PostgresSQLConnection(DbConnection conn) {
 
   @Override
   public SQLConnection setAutoCommit(boolean autoCommit, Handler<AsyncResult<Void>> handler) {
-    throw new UnsupportedOperationException();
+    inAutoCommit.set(autoCommit);
```

Comment:
no, if we follow the jdbc semantics, turning autocommit to false during a tx should commit the tx if i'm not mistaken